platform/command/transport_http.go

```diff
@@ -47,7 +47,7 @@ func EncodeError(ctx context.Context, err error, w http.ResponseWriter) {
 
 func decodeRequest(ctx context.Context, r *http.Request) (interface{}, error) {
 	var req newCommandRequest
-	err := json.NewDecoder(io.LimitReader(r.Body, 100000)).Decode(&req)
+	err := json.NewDecoder(io.LimitReader(r.Body, 1000000)).Decode(&req)
 	return req, err
```

Comment:
could you also change the return from 
```
return req, err
```

to 

```
return req, errors.Wrap(err, "decoding command request")
```

Reply:
Just updated